frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,134 @@
+import { execSync } from 'child_process';
+import { protractor, by, browser, ExpectedConditions as until } from 'protractor';
+import {
+  appHost,
+  checkLogs,
+  checkErrors,
+  testName,
+  retry,
+} from '../../../../integration-tests/protractor.conf';
+import {
+  installServerlessOperator,
+  firstKebabMenu,
+  machineSetsInput,
+  submitCount,
+  hardRefreshPage,
+} from '../views/serverless.view';
+import { switchPerspective, Perspective, sideHeader } from '../views/dev-perspective.view';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import * as catalogView from '../../../../integration-tests/views/catalog.view';
+import * as catalogPageView from '../../../../integration-tests/views/catalog-page.view';
+import * as operatorView from '../../../operator-lifecycle-manager/integration-tests/views/operator.view';
+import * as operatorHubView from '../../../operator-lifecycle-manager/integration-tests/views/operator-hub.view';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 5;
+
+describe('Serverless', async () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/k8s/cluster/projects`);
+    await switchPerspective(Perspective.Administrator);
+    expect(sideHeader.getText()).toContain('Administrator');
+  });
+
+  afterAll(() => {
+    // Set jasmine timeout interval back to the original value after these tests are done
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_DEFAULT_TIMEOUT_INTERVAL;
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('Increase the Machine sets count', async () => {
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Compute')));
+    await sidenavView.clickNavLink(['Compute', 'Machine Sets']);
+    await crudView.isLoaded();
+
+    await browser.wait(until.elementToBeClickable(firstKebabMenu));
+    await firstKebabMenu.click();
+    browser.wait(until.elementToBeClickable(crudView.actionForLabel('Edit Count')));
+    crudView.actionForLabel('Edit Count').click();
+
+    await browser.wait(until.elementToBeClickable(machineSetsInput));
+    await machineSetsInput.sendKeys(protractor.Key.chord(protractor.Key.CONTROL, 'a'));
+    await machineSetsInput.sendKeys('6');
+    await browser.wait(until.elementToBeClickable(submitCount));
+    await submitCount.click();
+    expect(true).toBeTruthy();
```

Comment:
what are you trying to check here ? `true` will always be `toBeTruthy`

Reply:
Actually this test will be extended to the knative-serving type of deployment. For now, it will be there. In the next PR, I will edit the same.

Reply:
we should have a valid assert, it doesn't make sense to have it block which will never fail